src/main/java/foxtrot/uniform/charlie/kilo/model/Ticket.java

```diff
@@ -0,0 +1,60 @@
+package foxtrot.uniform.charlie.kilo.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private MovieSession movieSession;
+    @OneToOne
```

Comment:
are you sure that here it is 1-1 relationship?

Reply:
I'm not sure about anything..........
But here I thought like one ticket could be only for one user since it's impossible that several user could have the same ticket.

Reply:
one user can buy many tickets, right?

Reply:
I though they couldn't.